engine/api/api.go

```diff
@@ -711,11 +682,8 @@ func (a *API) Serve(ctx context.Context) error {
 	migrate.Add(ctx, sdk.Migration{Name: "AddDefaultVCSNotifications", Release: "0.41.0", Automatic: true, ExecFunc: func(ctx context.Context) error {
```

Comment:
AddDefaultVCSNotifications could be removed, as RefactorGroupMembership must be executed on a release 0.44.0 min